<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(243, 248, 254)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(7, 148, 255);
	border-radius:5px;
	color: white;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>登      录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(240, 240, 240);
	border-radius:5px;
	color: grey;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>取      消</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Remenber">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
	color:rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoLogin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
	color:rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>自动登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserNameLine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	font: 11pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>输入电话号码</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="PassWordLine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	font: 11pt &quot;微软雅黑&quot;;
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>输入密码</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 26pt &quot;Microsoft YaHei UI&quot;;
color:rgb(0, 157, 255);</string>
    </property>
    <property name="text">
     <string>航班管理系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 120, 212)
</string>
    </property>
    <property name="text">
     <string>新用户注册</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="adLoginButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 120, 212)</string>
    </property>
    <property name="text">
     <string>管理员登录</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pwd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>CancelButton</tabstop>
  <tabstop>UserNameLine</tabstop>
  <tabstop>PassWordLine</tabstop>
  <tabstop>Remenber</tabstop>
  <tabstop>AutoLogin</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>adLoginButton</tabstop>
  <tabstop>RegisterButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
